brick-antispoof/src/antispoof.c

```diff
@@ -0,0 +1,235 @@
+/* Copyright 2015 Outscale SAS
+ *
+ * This file is part of Butterfly.
+ *
+ * Butterfly is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as published
+ * by the Free Software Foundation.
+ *
+ * Butterfly is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Butterfly.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <rte_config.h>
+#include <rte_ether.h>
+#include <rte_ip.h>
+#include <packetgraph/brick.h>
+#include <packetgraph/antispoof.h>
+#include <packetgraph/utils/bitmask.h>
+#include <packetgraph/utils/mac.h>
+
+struct pg_antispoof_arp {
+	/* Format of hardware address.  */
+	uint16_t ar_hrd;
+	/* Format of protocol address.  */
+	uint16_t ar_pro;
+	/* Length of hardware address.  */
+	uint8_t ar_hln;
+	/* Length of protocol address.  */
+	uint8_t ar_pln;
+	/* ARP opcode (command). */
+	uint16_t ar_op;
+	/* Sender hardware address. */
+	uint8_t sender_mac[ETHER_ADDR_LEN];
```

Comment:
Maybe you should have use ether_addr from rte_ether.h struct instead of an uint8_t